ci/actions/linux/ghcr_push.sh

```diff
@@ -2,10 +2,11 @@
 set -e
 
 scripts="$PWD/ci"
-if [[ "$GITHUB_WORKFLOW" = "Develop" ]]; then
+if [[ "$GITHUB_WORKFLOW" = "Develop Branch Dockers Deploy" ]]; then
     "$scripts"/custom-timeout.sh 30 docker push "ghcr.io/${GITHUB_REPOSITORY}/nano-env:base"
     "$scripts"/custom-timeout.sh 30 docker push "ghcr.io/${GITHUB_REPOSITORY}/nano-env:gcc"
     "$scripts"/custom-timeout.sh 30 docker push "ghcr.io/${GITHUB_REPOSITORY}/nano-env:clang"
+    "$scripts"/custom-timeout.sh 30 docker push "ghcr.io/${GITHUB_REPOSITORY}/nano-env:centos"
```

Comment:
What does this line fix?

Reply:
It does not fix anything. It just improves consistency.
It treats the centos docker the same way as other dockers.